From bd498f629813d8dd586a11007d777399ff1e2155 Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Mon, 4 Nov 2019 04:51:16 -0800
Subject: [PATCH] imx8/imx8m: Remove "fsl-" prefix from kernel DTB

Update the kernel DTB names for imx8 and imx8m platforms, remove the
"fsl-" prefix.

apply for 4.19.35_1.1.0 branch

Signed-off-by: Ye Li <ye.li@nxp.com>
---
 board/freescale/imx8dxl_phantom_mek/imx8dxl_phantom_mek.c      | 4 ++--
 board/freescale/imx8qm_mek/imx8qm_mek.c                        | 4 ++--
 board/freescale/imx8qxp_mek/imx8qxp_mek.c                      | 4 ++--
 configs/imx8dx_17x17_val_defconfig                             | 2 +-
 configs/imx8mm_ddr3l_val_defconfig                             | 2 +-
 configs/imx8mm_ddr4_evk_defconfig                              | 2 +-
 configs/imx8mm_ddr4_evk_nand_defconfig                         | 2 +-
 configs/imx8mm_ddr4_val_defconfig                              | 2 +-
 configs/imx8mm_evk_defconfig                                   | 2 +-
 configs/imx8mm_evk_fspi_defconfig                              | 2 +-
 configs/imx8mn_ddr4_evk_android_defconfig                      | 2 +-
 configs/imx8mn_ddr4_evk_android_dual_defconfig                 | 2 +-
 configs/imx8mn_ddr4_evk_android_trusty_defconfig               | 2 +-
 configs/imx8mn_ddr4_evk_android_trusty_dual_defconfig          | 2 +-
 configs/imx8mn_ddr4_evk_android_trusty_secure_unlock_defconfig | 2 +-
 configs/imx8mn_ddr4_evk_android_uuu_defconfig                  | 2 +-
 configs/imx8mn_ddr4_evk_defconfig                              | 2 +-
 configs/imx8mn_ddr4_evk_nom_defconfig                          | 2 +-
 configs/imx8mq_ddr3l_arm2_defconfig                            | 4 ++--
 configs/imx8mq_ddr4_arm2_defconfig                             | 2 +-
 configs/imx8mq_ddr4_arm2_nand_defconfig                        | 2 +-
 configs/imx8qm_ddr4_arm2_defconfig                             | 2 +-
 configs/imx8qm_lpddr4_arm2_android_defconfig                   | 2 +-
 configs/imx8qm_lpddr4_arm2_defconfig                           | 2 +-
 configs/imx8qm_lpddr4_arm2_fspi_defconfig                      | 2 +-
 configs/imx8qxp_17x17_val_defconfig                            | 2 +-
 configs/imx8qxp_ddr3_arm2_defconfig                            | 2 +-
 configs/imx8qxp_lpddr4_arm2_android_defconfig                  | 2 +-
 configs/imx8qxp_lpddr4_arm2_defconfig                          | 2 +-
 configs/imx8qxp_lpddr4_arm2_fspi_defconfig                     | 2 +-
 configs/imx8qxp_lpddr4_arm2_nand_defconfig                     | 2 +-
 include/configs/imx8dxl_phantom_mek.h                          | 6 +++---
 include/configs/imx8mm_evk.h                                   | 4 ++--
 include/configs/imx8mn_evk.h                                   | 4 ++--
 include/configs/imx8mq_aiy.h                                   | 2 +-
 include/configs/imx8mq_evk.h                                   | 6 +++---
 include/configs/imx8qm_arm2.h                                  | 2 +-
 include/configs/imx8qm_mek.h                                   | 6 +++---
 include/configs/imx8qxp_mek.h                                  | 6 +++---
 39 files changed, 53 insertions(+), 53 deletions(-)

diff --git a/board/freescale/imx8dxl_phantom_mek/imx8dxl_phantom_mek.c b/board/freescale/imx8dxl_phantom_mek/imx8dxl_phantom_mek.c
index decb526..f8f5c93 100644
--- a/board/freescale/imx8dxl_phantom_mek/imx8dxl_phantom_mek.c
+++ b/board/freescale/imx8dxl_phantom_mek/imx8dxl_phantom_mek.c
@@ -266,9 +266,9 @@ int board_late_init(void)
 
 	if (fdt_file && !strcmp(fdt_file, "undefined")) {
 		if (m4_boot)
-			env_set("fdt_file", "fsl-imx8dxl-phantom-mek-rpmsg.dtb");
+			env_set("fdt_file", "imx8dxl-phantom-mek-rpmsg.dtb");
 		else
-			env_set("fdt_file", "fsl-imx8dxl-phantom-mek.dtb");
+			env_set("fdt_file", "imx8dxl-phantom-mek.dtb");
 	}
 
 #ifdef CONFIG_ENV_IS_IN_MMC
diff --git a/board/freescale/imx8qm_mek/imx8qm_mek.c b/board/freescale/imx8qm_mek/imx8qm_mek.c
index 62bec8e..4e0bd76 100644
--- a/board/freescale/imx8qm_mek/imx8qm_mek.c
+++ b/board/freescale/imx8qm_mek/imx8qm_mek.c
@@ -433,9 +433,9 @@ int board_late_init(void)
 
 	if (fdt_file && !strcmp(fdt_file, "undefined")) {
 		if (m4_boot)
-			env_set("fdt_file", "fsl-imx8qm-mek-rpmsg.dtb");
+			env_set("fdt_file", "imx8qm-mek-rpmsg.dtb");
 		else
-			env_set("fdt_file", "fsl-imx8qm-mek.dtb");
+			env_set("fdt_file", "imx8qm-mek.dtb");
 	}
 
 #ifdef CONFIG_ENV_IS_IN_MMC
diff --git a/board/freescale/imx8qxp_mek/imx8qxp_mek.c b/board/freescale/imx8qxp_mek/imx8qxp_mek.c
index d9e3343..4298867 100644
--- a/board/freescale/imx8qxp_mek/imx8qxp_mek.c
+++ b/board/freescale/imx8qxp_mek/imx8qxp_mek.c
@@ -442,9 +442,9 @@ int board_late_init(void)
 
 	if (fdt_file && !strcmp(fdt_file, "undefined")) {
 		if (m4_boot)
-			env_set("fdt_file", "fsl-imx8qxp-mek-rpmsg.dtb");
+			env_set("fdt_file", "imx8qxp-mek-rpmsg.dtb");
 		else
-			env_set("fdt_file", "fsl-imx8qxp-mek.dtb");
+			env_set("fdt_file", "imx8qxp-mek.dtb");
 	}
 
 #ifdef CONFIG_ENV_IS_IN_MMC
diff --git a/configs/imx8dx_17x17_val_defconfig b/configs/imx8dx_17x17_val_defconfig
index 8b609fc..3df47e3 100644
--- a/configs/imx8dx_17x17_val_defconfig
+++ b/configs/imx8dx_17x17_val_defconfig
@@ -42,7 +42,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8dx-17x17-val"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8dx-17x17-val.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8dx-17x17-val.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8mm_ddr3l_val_defconfig b/configs/imx8mm_ddr3l_val_defconfig
index a2e8ddb..81e745c 100644
--- a/configs/imx8mm_ddr3l_val_defconfig
+++ b/configs/imx8mm_ddr3l_val_defconfig
@@ -22,7 +22,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mm-ddr3l-val"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mm-ddr3l-val.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mm-ddr3l-val.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mm_ddr4_evk_defconfig b/configs/imx8mm_ddr4_evk_defconfig
index 981474a..4394608 100644
--- a/configs/imx8mm_ddr4_evk_defconfig
+++ b/configs/imx8mm_ddr4_evk_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mm-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mm-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mm-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_GPIO=y
diff --git a/configs/imx8mm_ddr4_evk_nand_defconfig b/configs/imx8mm_ddr4_evk_nand_defconfig
index 1230d56..87bf123 100644
--- a/configs/imx8mm_ddr4_evk_nand_defconfig
+++ b/configs/imx8mm_ddr4_evk_nand_defconfig
@@ -22,7 +22,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mm-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mm-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mm-ddr4-evk.dtb"
 CONFIG_CMD_I2C=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_CACHE=y
diff --git a/configs/imx8mm_ddr4_val_defconfig b/configs/imx8mm_ddr4_val_defconfig
index aadc017..00f39be 100644
--- a/configs/imx8mm_ddr4_val_defconfig
+++ b/configs/imx8mm_ddr4_val_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mm-ddr4-val"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mm-ddr4-val.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mm-ddr4-val.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mm_evk_defconfig b/configs/imx8mm_evk_defconfig
index 478ddde..d4db86c 100644
--- a/configs/imx8mm_evk_defconfig
+++ b/configs/imx8mm_evk_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mm-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mm-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mm-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mm_evk_fspi_defconfig b/configs/imx8mm_evk_fspi_defconfig
index c68aa94..5dcf608 100644
--- a/configs/imx8mm_evk_fspi_defconfig
+++ b/configs/imx8mm_evk_fspi_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mm-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mm-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mm-evk.dtb"
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mn_ddr4_evk_android_defconfig b/configs/imx8mn_ddr4_evk_android_defconfig
index c41d150..d711b5c 100644
--- a/configs/imx8mn_ddr4_evk_android_defconfig
+++ b/configs/imx8mn_ddr4_evk_android_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_CMD_SF=y
diff --git a/configs/imx8mn_ddr4_evk_android_dual_defconfig b/configs/imx8mn_ddr4_evk_android_dual_defconfig
index cfabfd4..be77f5c 100644
--- a/configs/imx8mn_ddr4_evk_android_dual_defconfig
+++ b/configs/imx8mn_ddr4_evk_android_dual_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_CMD_SF=y
diff --git a/configs/imx8mn_ddr4_evk_android_trusty_defconfig b/configs/imx8mn_ddr4_evk_android_trusty_defconfig
index 13b2639..ba32356 100644
--- a/configs/imx8mn_ddr4_evk_android_trusty_defconfig
+++ b/configs/imx8mn_ddr4_evk_android_trusty_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mn_ddr4_evk_android_trusty_dual_defconfig b/configs/imx8mn_ddr4_evk_android_trusty_dual_defconfig
index 309e895..e3adea8 100644
--- a/configs/imx8mn_ddr4_evk_android_trusty_dual_defconfig
+++ b/configs/imx8mn_ddr4_evk_android_trusty_dual_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mn_ddr4_evk_android_trusty_secure_unlock_defconfig b/configs/imx8mn_ddr4_evk_android_trusty_secure_unlock_defconfig
index 8764404..5e17982 100644
--- a/configs/imx8mn_ddr4_evk_android_trusty_secure_unlock_defconfig
+++ b/configs/imx8mn_ddr4_evk_android_trusty_secure_unlock_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mn_ddr4_evk_android_uuu_defconfig b/configs/imx8mn_ddr4_evk_android_uuu_defconfig
index 3157570..e5f9e68 100644
--- a/configs/imx8mn_ddr4_evk_android_uuu_defconfig
+++ b/configs/imx8mn_ddr4_evk_android_uuu_defconfig
@@ -23,7 +23,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_CMD_SF=y
diff --git a/configs/imx8mn_ddr4_evk_defconfig b/configs/imx8mn_ddr4_evk_defconfig
index 88e6c53..3291e12 100644
--- a/configs/imx8mn_ddr4_evk_defconfig
+++ b/configs/imx8mn_ddr4_evk_defconfig
@@ -22,7 +22,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_CMD_SF=y
diff --git a/configs/imx8mn_ddr4_evk_nom_defconfig b/configs/imx8mn_ddr4_evk_nom_defconfig
index 9d70fbc..1bdc00f 100644
--- a/configs/imx8mn_ddr4_evk_nom_defconfig
+++ b/configs/imx8mn_ddr4_evk_nom_defconfig
@@ -22,7 +22,7 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_CMD_FAT=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8mn-ddr4-evk"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mn-ddr4-evk.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mn-ddr4-evk.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
diff --git a/configs/imx8mq_ddr3l_arm2_defconfig b/configs/imx8mq_ddr3l_arm2_defconfig
index 2d45311..b5bc197 100644
--- a/configs/imx8mq_ddr3l_arm2_defconfig
+++ b/configs/imx8mq_ddr3l_arm2_defconfig
@@ -4,7 +4,7 @@ CONFIG_SYS_TEXT_BASE=0x40200000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
 CONFIG_TARGET_IMX8MQ_DDR3L_ARM2=y
 CONFIG_SPL_SERIAL_SUPPORT=y
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mq-ddr3l-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mq-ddr3l-arm2.dtb"
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_SPL=y
@@ -80,4 +80,4 @@ CONFIG_SPI_FLASH_GIGADEVICE=y
 CONFIG_SF_DEFAULT_BUS=0
 CONFIG_SF_DEFAULT_CS=1
 CONFIG_SF_DEFAULT_SPEED=40000000
-CONFIG_SF_DEFAULT_MODE=0
\ No newline at end of file
+CONFIG_SF_DEFAULT_MODE=0
diff --git a/configs/imx8mq_ddr4_arm2_defconfig b/configs/imx8mq_ddr4_arm2_defconfig
index c4d810c..37fa43f 100644
--- a/configs/imx8mq_ddr4_arm2_defconfig
+++ b/configs/imx8mq_ddr4_arm2_defconfig
@@ -4,7 +4,7 @@ CONFIG_SYS_TEXT_BASE=0x40200000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
 CONFIG_TARGET_IMX8MQ_DDR4_ARM2=y
 CONFIG_SPL_SERIAL_SUPPORT=y
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mq-ddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mq-ddr4-arm2.dtb"
 CONFIG_NR_DRAM_BANKS=2
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_SPL=y
diff --git a/configs/imx8mq_ddr4_arm2_nand_defconfig b/configs/imx8mq_ddr4_arm2_nand_defconfig
index a3f72ec..08f2c23 100644
--- a/configs/imx8mq_ddr4_arm2_nand_defconfig
+++ b/configs/imx8mq_ddr4_arm2_nand_defconfig
@@ -4,7 +4,7 @@ CONFIG_SYS_TEXT_BASE=0x40200000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
 CONFIG_TARGET_IMX8MQ_DDR4_ARM2=y
 CONFIG_SPL_SERIAL_SUPPORT=y
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8mq-ddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8mq-ddr4-arm2.dtb"
 CONFIG_NR_DRAM_BANKS=2
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_SPL=y
diff --git a/configs/imx8qm_ddr4_arm2_defconfig b/configs/imx8qm_ddr4_arm2_defconfig
index f66dc11..eae8c9a 100644
--- a/configs/imx8qm_ddr4_arm2_defconfig
+++ b/configs/imx8qm_ddr4_arm2_defconfig
@@ -42,7 +42,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qm-ddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qm-ddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qm-ddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qm_lpddr4_arm2_android_defconfig b/configs/imx8qm_lpddr4_arm2_android_defconfig
index 1c019ce..919e168 100644
--- a/configs/imx8qm_lpddr4_arm2_android_defconfig
+++ b/configs/imx8qm_lpddr4_arm2_android_defconfig
@@ -41,7 +41,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qm-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qm-lpddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qm-lpddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qm_lpddr4_arm2_defconfig b/configs/imx8qm_lpddr4_arm2_defconfig
index f6e1c44..f3265ca 100644
--- a/configs/imx8qm_lpddr4_arm2_defconfig
+++ b/configs/imx8qm_lpddr4_arm2_defconfig
@@ -42,7 +42,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qm-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qm-lpddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qm-lpddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qm_lpddr4_arm2_fspi_defconfig b/configs/imx8qm_lpddr4_arm2_fspi_defconfig
index 7ae5257..0be5d91 100644
--- a/configs/imx8qm_lpddr4_arm2_fspi_defconfig
+++ b/configs/imx8qm_lpddr4_arm2_fspi_defconfig
@@ -44,7 +44,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qm-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qm-lpddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qm-lpddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qxp_17x17_val_defconfig b/configs/imx8qxp_17x17_val_defconfig
index a268367..d0c6c87 100644
--- a/configs/imx8qxp_17x17_val_defconfig
+++ b/configs/imx8qxp_17x17_val_defconfig
@@ -42,7 +42,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qxp-17x17-val"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qxp-17x17-val.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qxp-17x17-val.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qxp_ddr3_arm2_defconfig b/configs/imx8qxp_ddr3_arm2_defconfig
index 34958e4..779b493 100644
--- a/configs/imx8qxp_ddr3_arm2_defconfig
+++ b/configs/imx8qxp_ddr3_arm2_defconfig
@@ -42,7 +42,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qxp-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qxp-ddr3l-val.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qxp-ddr3l-val.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qxp_lpddr4_arm2_android_defconfig b/configs/imx8qxp_lpddr4_arm2_android_defconfig
index 1508d6c..fc01a1c 100644
--- a/configs/imx8qxp_lpddr4_arm2_android_defconfig
+++ b/configs/imx8qxp_lpddr4_arm2_android_defconfig
@@ -41,7 +41,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qxp-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qxp-lpddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qxp-lpddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qxp_lpddr4_arm2_defconfig b/configs/imx8qxp_lpddr4_arm2_defconfig
index c56bae2..4841c71 100644
--- a/configs/imx8qxp_lpddr4_arm2_defconfig
+++ b/configs/imx8qxp_lpddr4_arm2_defconfig
@@ -42,7 +42,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qxp-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qxp-lpddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qxp-lpddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qxp_lpddr4_arm2_fspi_defconfig b/configs/imx8qxp_lpddr4_arm2_fspi_defconfig
index 4b75881..66b8922 100644
--- a/configs/imx8qxp_lpddr4_arm2_fspi_defconfig
+++ b/configs/imx8qxp_lpddr4_arm2_fspi_defconfig
@@ -44,7 +44,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qxp-lpddr4-arm2"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qxp-lpddr4-arm2.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qxp-lpddr4-arm2.dtb"
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_SPL_DM=y
 CONFIG_SPL_CLK=y
diff --git a/configs/imx8qxp_lpddr4_arm2_nand_defconfig b/configs/imx8qxp_lpddr4_arm2_nand_defconfig
index 980ed44..13f083d 100644
--- a/configs/imx8qxp_lpddr4_arm2_nand_defconfig
+++ b/configs/imx8qxp_lpddr4_arm2_nand_defconfig
@@ -39,7 +39,7 @@ CONFIG_CMD_CACHE=y
 CONFIG_CMD_FAT=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-imx8qxp-lpddr4-arm2-gpmi-nand"
-CONFIG_DEFAULT_FDT_FILE="fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dtb"
+CONFIG_DEFAULT_FDT_FILE="imx8qxp-lpddr4-arm2-gpmi-nand.dtb"
 CONFIG_ENV_IS_IN_NAND=y
 CONFIG_SPL_DM=y
 CONFIG_CLK_IMX8=y
diff --git a/include/configs/imx8dxl_phantom_mek.h b/include/configs/imx8dxl_phantom_mek.h
index 15bf04c..d94ad9f 100644
--- a/include/configs/imx8dxl_phantom_mek.h
+++ b/include/configs/imx8dxl_phantom_mek.h
@@ -109,11 +109,11 @@
 
 #define JAILHOUSE_ENV \
 	"jh_mmcboot=" \
-		"setenv fdt_file fsl-imx8qxp-mek-root.dtb;"\
+		"setenv fdt_file imx8qxp-mek-root.dtb;"\
 		"setenv boot_os 'scu_rm dtb ${fdt_addr}; booti ${loadaddr} - ${fdt_addr};'; " \
 		"run mmcboot; \0" \
 	"jh_netboot=" \
-		"setenv fdt_file fsl-imx8qxp-mek-root.dtb;"\
+		"setenv fdt_file imx8qxp-mek-root.dtb;"\
 		"setenv boot_os 'scu_rm dtb ${fdt_addr}; booti ${loadaddr} - ${fdt_addr};'; " \
 		"run netboot; \0"
 
@@ -122,7 +122,7 @@
             "xenlinux_bootargs= \0" \
             "xenlinux_console=hvc0 earlycon=xen\0" \
             "xenlinux_addr=0x9e000000\0" \
-	    "dom0fdt_file=fsl-imx8qxp-mek-dom0.dtb\0" \
+	    "dom0fdt_file=imx8qxp-mek-dom0.dtb\0" \
             "xenboot_common=" \
                 "${get_cmd} ${loadaddr} xen;" \
                 "${get_cmd} ${fdt_addr} ${dom0fdt_file};" \
diff --git a/include/configs/imx8mm_evk.h b/include/configs/imx8mm_evk.h
index 9255cfe..229bd60 100644
--- a/include/configs/imx8mm_evk.h
+++ b/include/configs/imx8mm_evk.h
@@ -121,12 +121,12 @@
  */
 #define JAILHOUSE_ENV \
 	"jh_clk= \0 " \
-	"jh_mmcboot=mw 0x303d0518 0xff; setenv fdt_file fsl-imx8mm-evk-root.dtb;" \
+	"jh_mmcboot=mw 0x303d0518 0xff; setenv fdt_file imx8mm-evk-root.dtb;" \
 		"setenv jh_clk clk_ignore_unused; " \
 			   "if run loadimage; then " \
 				   "run mmcboot; " \
 			   "else run jh_netboot; fi; \0" \
-	"jh_netboot=mw 0x303d0518 0xff; setenv fdt_file fsl-imx8mm-evk-root.dtb; setenv jh_clk clk_ignore_unused; run netboot; \0 "
+	"jh_netboot=mw 0x303d0518 0xff; setenv fdt_file imx8mm-evk-root.dtb; setenv jh_clk clk_ignore_unused; run netboot; \0 "
 
 
 #define CONFIG_MFG_ENV_SETTINGS \
diff --git a/include/configs/imx8mn_evk.h b/include/configs/imx8mn_evk.h
index ac91d5b..4d862fb 100644
--- a/include/configs/imx8mn_evk.h
+++ b/include/configs/imx8mn_evk.h
@@ -114,12 +114,12 @@
 
 #define JAILHOUSE_ENV \
 	"jh_clk= \0 " \
-	"jh_mmcboot=mw 0x303d0518 0xff; setenv fdt_file fsl-imx8mn-ddr4-evk-root.dtb;" \
+	"jh_mmcboot=mw 0x303d0518 0xff; setenv fdt_file imx8mn-ddr4-evk-root.dtb;" \
 		"setenv jh_clk clk_ignore_unused; " \
 			   "if run loadimage; then " \
 				   "run mmcboot; " \
 			   "else run jh_netboot; fi; \0" \
-	"jh_netboot=mw 0x303d0518 0xff; setenv fdt_file fsl-imx8mn-ddr4-evk-root.dtb; setenv jh_clk clk_ignore_unused; run netboot; \0 "
+	"jh_netboot=mw 0x303d0518 0xff; setenv fdt_file imx8mn-ddr4-evk-root.dtb; setenv jh_clk clk_ignore_unused; run netboot; \0 "
 
 #define CONFIG_MFG_ENV_SETTINGS \
 	CONFIG_MFG_ENV_SETTINGS_DEFAULT \
diff --git a/include/configs/imx8mq_aiy.h b/include/configs/imx8mq_aiy.h
index b0b7c95..4b486d0 100644
--- a/include/configs/imx8mq_aiy.h
+++ b/include/configs/imx8mq_aiy.h
@@ -122,7 +122,7 @@
 	"fdt_addr=0x43000000\0"			\
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fdt=try\0" \
-	"fdt_file=fsl-imx8mq-phanbell.dtb\0" \
+	"fdt_file=imx8mq-phanbell.dtb\0" \
 	"initrd_addr=0x43800000\0"		\
 	"initrd_high=0xffffffffffffffff\0" \
 	"mmcdev="__stringify(CONFIG_SYS_MMC_ENV_DEV)"\0" \
diff --git a/include/configs/imx8mq_evk.h b/include/configs/imx8mq_evk.h
index 4b04801..f153337 100644
--- a/include/configs/imx8mq_evk.h
+++ b/include/configs/imx8mq_evk.h
@@ -102,12 +102,12 @@
  */
 #define JAILHOUSE_ENV \
 	"jh_clk= \0 " \
-	"jh_mmcboot=setenv fdt_file fsl-imx8mq-evk-root.dtb; " \
+	"jh_mmcboot=setenv fdt_file imx8mq-evk-root.dtb; " \
 		"setenv jh_clk clk_ignore_unused; " \
 			   "if run loadimage; then " \
 				   "run mmcboot; " \
 			   "else run jh_netboot; fi; \0" \
-	"jh_netboot=setenv fdt_file fsl-imx8mq-evk-root.dtb; setenv jh_clk clk_ignore_unused; run netboot; \0 "
+	"jh_netboot=setenv fdt_file imx8mq-evk-root.dtb; setenv jh_clk clk_ignore_unused; run netboot; \0 "
 
 #define CONFIG_MFG_ENV_SETTINGS \
 	CONFIG_MFG_ENV_SETTINGS_DEFAULT \
@@ -126,7 +126,7 @@
 	"fdt_addr=0x43000000\0"			\
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fdt=try\0" \
-	"fdt_file=fsl-imx8mq-evk.dtb\0" \
+	"fdt_file=imx8mq-evk.dtb\0" \
 	"initrd_addr=0x43800000\0"		\
 	"initrd_high=0xffffffffffffffff\0" \
 	"mmcdev="__stringify(CONFIG_SYS_MMC_ENV_DEV)"\0" \
diff --git a/include/configs/imx8qm_arm2.h b/include/configs/imx8qm_arm2.h
index 7b470a3..362b914 100644
--- a/include/configs/imx8qm_arm2.h
+++ b/include/configs/imx8qm_arm2.h
@@ -135,7 +135,7 @@
             "xenlinux_bootargs= \0" \
             "xenlinux_console=hvc0 earlycon=xen\0" \
             "xenlinux_addr=0x9e000000\0" \
-	    "dom0fdt_file=fsl-imx8qm-lpddr4-arm2-dom0.dtb\0" \
+	    "dom0fdt_file=imx8qm-lpddr4-arm2-dom0.dtb\0" \
             "xenboot_common=" \
                 "${get_cmd} ${loadaddr} xen;" \
                 "${get_cmd} ${fdt_addr} ${dom0fdt_file};" \
diff --git a/include/configs/imx8qm_mek.h b/include/configs/imx8qm_mek.h
index b85e51d..474d4d6 100644
--- a/include/configs/imx8qm_mek.h
+++ b/include/configs/imx8qm_mek.h
@@ -111,11 +111,11 @@
 
 #define JAILHOUSE_ENV \
 	"jh_mmcboot=" \
-		"setenv fdt_file fsl-imx8qm-mek-root.dtb;"\
+		"setenv fdt_file imx8qm-mek-root.dtb;"\
 		"setenv boot_os 'scu_rm dtb ${fdt_addr}; booti ${loadaddr} - ${fdt_addr};'; " \
 		"run mmcboot; \0" \
 	"jh_netboot=" \
-		"setenv fdt_file fsl-imx8qm-mek-root.dtb;"\
+		"setenv fdt_file imx8qm-mek-root.dtb;"\
 		"setenv boot_os 'scu_rm dtb ${fdt_addr}; booti ${loadaddr} - ${fdt_addr};'; " \
 		"run netboot; \0"
 
@@ -125,7 +125,7 @@
             "xenlinux_bootargs= \0" \
             "xenlinux_console=hvc0 earlycon=xen\0" \
             "xenlinux_addr=0x9e000000\0" \
-	    "dom0fdt_file=fsl-imx8qm-mek-dom0.dtb\0" \
+	    "dom0fdt_file=imx8qm-mek-dom0.dtb\0" \
             "xenboot_common=" \
                 "${get_cmd} ${loadaddr} xen;" \
                 "${get_cmd} ${fdt_addr} ${dom0fdt_file};" \
diff --git a/include/configs/imx8qxp_mek.h b/include/configs/imx8qxp_mek.h
index ffffbae..24f8138 100644
--- a/include/configs/imx8qxp_mek.h
+++ b/include/configs/imx8qxp_mek.h
@@ -104,11 +104,11 @@
 
 #define JAILHOUSE_ENV \
 	"jh_mmcboot=" \
-		"setenv fdt_file fsl-imx8qxp-mek-root.dtb;"\
+		"setenv fdt_file imx8qxp-mek-root.dtb;"\
 		"setenv boot_os 'scu_rm dtb ${fdt_addr}; booti ${loadaddr} - ${fdt_addr};'; " \
 		"run mmcboot; \0" \
 	"jh_netboot=" \
-		"setenv fdt_file fsl-imx8qxp-mek-root.dtb;"\
+		"setenv fdt_file imx8qxp-mek-root.dtb;"\
 		"setenv boot_os 'scu_rm dtb ${fdt_addr}; booti ${loadaddr} - ${fdt_addr};'; " \
 		"run netboot; \0"
 
@@ -117,7 +117,7 @@
             "xenlinux_bootargs= \0" \
             "xenlinux_console=hvc0 earlycon=xen\0" \
             "xenlinux_addr=0x9e000000\0" \
-	    "dom0fdt_file=fsl-imx8qxp-mek-dom0.dtb\0" \
+	    "dom0fdt_file=imx8qxp-mek-dom0.dtb\0" \
             "xenboot_common=" \
                 "${get_cmd} ${loadaddr} xen;" \
                 "${get_cmd} ${fdt_addr} ${dom0fdt_file};" \
-- 
2.7.4

